app/uk/gov/hmrc/hmrcemailrenderer/templates/amls/amlsNotificationReceived.scala.html

```diff
@@ -1,38 +1,47 @@
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
 @(params: Map[String, Any])
 @uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "You have a new message from HMRC") {
 
-    <p style="margin: 0 0 30px; font-size: 19px;">The message is about your Anti-Money Laundering Supervision (AMLS). It is only visible from inside your AMLS account.</p>
+    <p style="margin: 0 0 30px;
+        font-size: 19px;">
+        The message is about your Anti-Money Laundering Supervision (AMLS). It is only visible from inside your AMLS account.</p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;">To read your message either:</p>
+    <p style="margin: 0 0 30px;
+        font-size: 19px;">To read your message either:</p>
 
-    <ul style="margin: 0 0 30px; font-size: 19px;"><li>Sign in to your AMLS account using the correct username and password</li>
-    <li>Sign in to your Business Tax Account and then open your AMLS account (under 'Other services')</li>
-    <li>Use the link to your AMLS account we sent you when you signed up</li></ul>
+    <ul style="margin: 0 0 30px;
+        font-size: 19px;"><li>Sign in to your AMLS account using the correct username and password</li>
+        <li>Sign in to your Business Tax Account and then open your AMLS account (under 'Other services')</li>
+        <li>Use the link to your AMLS account we sent you when you signed up</li></ul>
 
-    <p style="margin: 0 0 30px; font-size: 19px;">For security reasons we cannot include any links on this page.</p>
+    <p style="margin: 0 0 30px;
+        font-size: 19px;">For security reasons we cannot include any links on this page.</p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;"><h2>Why you got this email</h2></p>
+    <p style="margin: 0 0 30px;
+        font-size: 19px;"><h2>Why you got this email</h2></p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;">You gave this contact email address when you signed up for Anti-Money Laundering Supervision.</p>
+    <p style="margin: 0 0 30px;
+        font-size: 19px;">
+        You gave this contact email address when you signed up for Anti-Money Laundering Supervision.</p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;">This means we send you an email to let you know you have a new message in your account.</p>
+    <p style="margin: 0 0 30px;
+        font-size: 19px;">This means we send you an email to let you know you have a new message in your account.</p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;">From HMRC Anti-Money Laundering Supervision</p>
+    <p style="margin: 0 0 30px;
+        font-size: 19px;">From HMRC Anti-Money Laundering Supervision</p>
```

Comment:
This doesn't look like our code - how come it has changed?

Reply:
this is the result of their requirement to use scala formatter. Without this I could not pass checks